README.md

```diff
@@ -5,259 +5,77 @@
-
-- [Try out Label Studio](#try-out-label-studio)
-- [What you get from Label Studio](#what-you-get-from-label-studio)
-- [Included templates for labeling data in Label Studio](#included-templates-for-labeling-data-in-label-studio)
-- [Set up machine learning models with Label Studio](#set-up-machine-learning-models-with-Label-Studio)
-- [Integrate Label Studio with your existing tools](#integrate-label-studio-with-your-existing-tools)
-
-![Gif of Label Studio annotating different types of data](https://raw.githubusercontent.com/heartexlabs/label-studio/master/images/annotation_examples.gif)
-
-Have a custom dataset? You can customize Label Studio to fit your needs. Read an [introductory blog post](https://towardsdatascience.com/introducing-label-studio-a-swiss-army-knife-of-data-labeling-140c1be92881) to learn more. 
-
-## Try out Label Studio
-
-Install Label Studio locally, or deploy it in a cloud instance. [Or, sign up for a free trial of our Enterprise edition.](https://heartex.com/free-trial).
-
-- [Install locally with Docker](#install-locally-with-docker)
-- [Run with Docker Compose (Label Studio + Nginx + PostgreSQL)](#run-with-docker-compose)
-- [Install locally with pip](#install-locally-with-pip)
-- [Install locally with poetry](#install-locally-with-poetry)
-- [Install locally with Anaconda](#install-locally-with-anaconda)
-- [Install for local development](#install-for-local-development)
-- [Deploy in a cloud instance](#deploy-in-a-cloud-instance)
-
-### Install locally with Docker
-Official Label Studio docker image is [here](https://hub.docker.com/r/heartexlabs/label-studio) and it can be downloaded with `docker pull`. 
-Run Label Studio in a Docker container and access it at `http://localhost:8080`.
-
-
-```bash
-docker pull heartexlabs/label-studio:latest
-docker run -it -p 8080:8080 -v $(pwd)/mydata:/label-studio/data heartexlabs/label-studio:latest
+For the official documentation we refer to the parent label studio [repository](https://github.com/heartexlabs/label-studio).
+
+# Deployment Guide
+The below is a guide on how to deploy the label studio instance including their ml-backends. The deployment is meant for an EC2 (or other VM) and we assume that the infrastructure has already been set up.
+
+The deployment steps are:
+1. Adjust the docker-compose.yml files for label-studio and label-studio-ml-backend to contain the right volumes.
+2. Build and run label-studio docker and label-studio-ml-backend instances
+3. Setup your label studio instance
+
+## 1. Adjusting Docker Compose
+Adjust the docker-compose files to:
+1. Mount the directory with the data to the `app` service in label studio as well as in all ml-backend services.
+The right structure is something like `/home/ubuntu/data:/label-studio/files`. `/label-studio/files` needs to have a directory called `evaluation` containing the pdfs and `test_png` containing all corresponding png files.
```

Comment:
Be more explicit about where this needs to be changes. If I understand correctly, we need to adjust one setting of the docker-compose file in this repository, and two places in the docker-compose file from the label-studio-ml-backend repository (stratigraphy-ml-backend and text_extractor services)?
Also, would it be possible to use an environment variable to configure that location of the local data directory that needs to be mounted? The we don't need to be putting local directories into a docker-compose file that is tracked by git.

Reply:
We can specify the data location with environment variables. We'll still need to set it though.

I'll make a suggestion.

Reply:
See the documentation now. Just specifying environment variables is not that easy. One reason is, that the variables are not just passed to the tmux shell (which is a good thing as then the shell session is really independent of other shell sessions).

The recommended way by docker is to use a .env file that is read by default from docker compose. But that means, we still have to specify two .env files.